src/DependencyInjection/MvoContaoFacebookImportExtension.php

```diff
@@ -33,8 +33,8 @@ class MvoContaoFacebookImportExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('services.yml');
+        $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../../config'));
+        $loader->load('services.yaml');
```

Comment:
This change seems unrelated.

Reply:
see: https://github.com/m-vo/contao-facebook-import/pull/33